### PR TITLE
Fix example

### PR DESCRIPTION
**Pull request by huningxin**
There are syntax error and lint errors of the example. This PR fixes them. 

The idea is to keep aligned with the webnn example update as https://github.com/webmachinelearning/webnn-samples/pull/19. Here is a preview version: https://huningxin.github.io/webnn-samples/code/?example=simple_graph.js

@anssiko @wchao1115 @pyu10055 , please take a look. Thanks!

***
<a href="https://pr-preview.s3.amazonaws.com/huningxin/webnn/pull/104.html" title="Last updated on Oct 28, 2020, 2:48 PM UTC (c94c0a4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/104/058a81a...huningxin:c94c0a4.html" title="Last updated on Oct 28, 2020, 2:48 PM UTC (c94c0a4)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1175,7 +1175,7 @@ const intermediateOutput2 = builder.add(constant2, input2);
 const output = builder.mul(intermediateOutput1, intermediateOutput2);
 
 // Create the model by identifying the outputs.
-const model = builder.createModel({'output', output});
+const model = builder.createModel({output});
```
Comment (huningxin) [inline index.bs:1178]: syntax error

File: index.bs [hunk 2]
```diff
@@ -1184,7 +1184,7 @@ The following code compiles the model by prioritizing lower level of power consu
 over time. This option could be particularly useful for long-running models.
 <pre highlight="js">
 // Compile the constructed model.
-const compilation = await model.compile({ powerPreference: 'low-power' });
+const compilation = await model.compile({powerPreference: 'low-power'});
```
Comment (huningxin) [inline index.bs:1187]: eslint error: "There should be no space after '{'"

File: index.bs [hunk 3]
```diff
@@ -1184,7 +1184,7 @@ The following code compiles the model by prioritizing lower level of power consu
 over time. This option could be particularly useful for long-running models.
 <pre highlight="js">
 // Compile the constructed model.
-const compilation = await model.compile({ powerPreference: 'low-power' });
```
Comment (huningxin) [inline index.bs:1187]: eslint error: "There should be no space after '{'"

File: index.bs [hunk 4]
```diff
@@ -1196,7 +1196,11 @@ const inputBuffer1 = new Float32Array(TENSOR_SIZE).fill(1);
 const inputBuffer2 = new Float32Array(TENSOR_SIZE).fill(1);
 
 // Asynchronously execute the compiled model with the specified inputs.
-let outputs = await compilation.compute({'input1': {buffer: inputBuffer1}, 'input2': {buffer: inputBuffer2}});
```
Comment (huningxin) [inline index.bs:1199]: eslint error: exceeds max-len (80).

**Comment by huningxin** [comment]
Fixed the conflicts. Please take another look. Thanks!

**Review by anssiko — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (huningxin) [reply index.bs:1190]: I use the [shorthand property names (ES2015)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)

File: index.bs (hunk 3 above)
Comment (huningxin) [reply index.bs:1199]: The related eslint rule is [object-curly-spacing](https://eslint.org/docs/2.0.0/rules/object-curly-spacing) that disallows spaces inside of curly braces by default.

File: index.bs (hunk 4 above)
Comment (huningxin) [reply index.bs:1211]: The related eslint rule is [max-len](https://eslint.org/docs/rules/max-len#enforce-a-maximum-line-length-max-len) whose default value is 80.

**Comment by huningxin** [comment]
@wchao1115 , I found you are using spaces inside curly braces. Are you fine with this change?

**Comment by huningxin** [comment]
Rebased to ToT and improved the example by adding the log message.

**Comment by huningxin** [comment]
With the approval, I am going to merge this PR. Thanks!

**Pull request merged**